src/OrchardCore.Modules/OrchardCore.ContentFields/GraphQL/Types/UserPickerFieldQueryObjectType.cs

```diff
@@ -0,0 +1,71 @@
+using System.Collections.Generic;
+using System.Linq;
+using GraphQL;
+using GraphQL.DataLoader;
+using GraphQL.Types;
+using Microsoft.Extensions.DependencyInjection;
+using OrchardCore.Apis.GraphQL;
+using OrchardCore.ContentFields.Fields;
+using OrchardCore.ContentManagement;
+using OrchardCore.Users.GraphQL;
+using OrchardCore.Users.Indexes;
+using OrchardCore.Users.Models;
+using YesSql;
+using YesSql.Services;
+
+namespace OrchardCore.ContentFields.GraphQL
+{
+    public class UserPickerFieldQueryObjectType : ObjectGraphType<UserPickerField>
+    {
+        public UserPickerFieldQueryObjectType()
+        {
+            Name = nameof(UserPickerField);
+
+            Field<ListGraphType<StringGraphType>, IEnumerable<string>>("userIds")
+                .Description("user ids")
+                .PagingArguments()
+                .Resolve(x =>
+                {
+                    return x.Page(x.Source.UserIds);
+                });
+
+            Field<ListGraphType<UserType>, IEnumerable<User>>("users")
+                .Description("the user items")
+                .PagingArguments()
+                .ResolveAsync(x =>
+                {
+                    var userLoader = GetOrAddUserProfileByIdDataLoader(x);
+                    return userLoader.LoadAsync(x.Page(x.Source.UserIds)).Then(itemResultSet =>
+                    {
+                        return itemResultSet.SelectMany(x => x);
+                    });
+                });
+
+            Field<UserType, User>("user")
+                .Description("the first user")
+                .ResolveAsync(x =>
+                {
+                    var userLoader = GetOrAddUserProfileByIdDataLoader(x);
+                    return userLoader.LoadAsync(x.Source.UserIds.FirstOrDefault()).Then(itemResultSet =>
+                    {
+                        return itemResultSet.FirstOrDefault();
+                    });
+                });
```

Comment:
Do we really need this field?

Reply:
Why not? It can record the IDs of all content items and save them in a collection, and then query them in bulk from the database if the query need that field

Reply:
Oh sorry, I may have misunderstood you, you said 'first user', I think it should be useful in most cases, at least in js clients we can just use it without checking whether the array is empty first

``` js
const displayText=blog.author.firstUser?.displayText

const displayText2=blog.author?[0].displayText

```

Reply:
Well, it's a bug left over from the previous version of the code, which I did in loader. Items that don't have this field return empty references, so I figured out a way to add a `firstUser` field 😅

![image](https://github.com/OrchardCMS/OrchardCore/assets/15613121/5eef9c9c-344b-4726-a2a2-7a29a46e7226)

Reply:
The firstUser field has been removed.